install_files/ansible-base/usr.sbin.apache2

```diff
@@ -188,6 +196,8 @@
   /var/www/securedrop/worker.py r,
   /var/www/securedrop/worker.pyc rw,
   /var/www/source.wsgi r,
+  /var/www/.gnupg/ rw,
```

Comment:
Where does this come from? SecureDrop's key directory is `/var/lib/securedrop/keys`.